Button/button.css

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital@1&family=Ubuntu:wght@300&display=swap');
+body {
+    font-family: 'Source Sans Pro', sans-serif;
+    font-family: 'Ubuntu', sans-serif;
+}
+
+:root {
+    --primary-red: #DC2626;
+    --primary-green: #34D399;
+    --primary-yellow: #FBBF24;
+    --primary-blue: #2563EB;
+    --secondary-red: #F87171;
+    --secondary-green: #A7F3D0;
+    --secondary-yellow: #FDE68A;
+    --secondary-blue: #60A5FA;
+    --white: white;
```

Comment:
Suggestion:
Please use hex code values for consistency.

Reply:
Thanks for the review. I have made the changes.